docs/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "dev": "pnpm gen-locale && vitepress dev .",
-    "build": "NODE_ENV=production && vitepress build . && tsx .vitepress/build/rebuild-pwa.ts",
+    "build": "NODE_ENV=production && pnpm gen-locale && vitepress build . && tsx .vitepress/build/rebuild-pwa.ts",
```

Comment:
The [error log](https://github.com/element-plus/element-plus/actions/runs/4901540822/jobs/8752881481#step:14:20) shows that some translation file contains dead links. It might be related to https://crowdin.com/translate/element-plus/889/en-es?filter=basic&value=0#q=91813

Reply:
Yes, that's what caused the problem. 👍